src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,159 @@
+                break;
+            }
+            deadlineDescription = deadlineDescription + " " + words[i];
+        }
+        Deadlines newDeadline = new Deadlines(deadlineDescription.substring(1), by);
+        return newDeadline;
+    }
+
+    public static Events createNewEvent(String[] words) {
+        int eventIndex = 0;
+        String timeAllocation = "";
+        String eventDescription = "";
+        for(int i = 1 ; i < words.length ; i ++) {
+            if(words[i].equals("/at")) {
+                eventIndex = i + 1;
+                for(int j = eventIndex ; j < words.length ; j++) {
+                    timeAllocation = timeAllocation + " " + words[j];
+                }
+                break;
+            }
+            eventDescription = eventDescription + " " + words[i];
+        }
+        Events newEvent = new Events(eventDescription.substring(1), timeAllocation.substring(1));
+        return newEvent;
+    }
+
+    public static int distinguishCommand(String command , Task[] list, int size) {
+        //split into word array
+        String[] words = command.split(" ");
+        String firstWord = words[0];
+        int taskCount = size;
+
+        //determine command
+        switch (firstWord) {
+        case "list":
+            printList(list, size);
+            break;
+        case "done":
+            try {
+                markTaskDone(list, words[1]);
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println("    OOPS!!! The description of a done cannot be empty.");
+            }
+            break;
+        case "todo":
+            try {
+                ToDos newToDo = new ToDos(command.substring(5));
+                addTask(list, newToDo, taskCount);
+                taskCount ++;
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println("    OOPS!!! The description of a todo cannot be empty.");
+            }
+            break;
+        case "deadline":
+            try {
+                Deadlines newDeadline = createNewDeadline(words);
+                addTask(list, newDeadline, size);
+                taskCount ++;
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println("    OOPS!!! The description of a deadline cannot be empty.");
+            }
+            break;
+        case "event":
+            //extract out event description and timeAllocation
+            try {
+                Events newEvent = createNewEvent(words);
+                addTask(list, newEvent, size);
+                taskCount ++;
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println("    OOPS!!! The description of a event cannot be empty.");
+            }
+            break;
+        default:
+                System.out.println("    OOPS!!! I'm sorry, but I don't know what that means :-(");
+
+            break;
+        }
+
+        return taskCount;
+    }
+
+
+
```

Comment:
Suggest to leave only one new line instead of 3, so as to maintain a consistent code format.